os/fs/smartfs/smartfs_utils.c

```diff
@@ -1829,7 +1829,7 @@ int smartfs_examine_sector(struct smartfs_mountpt_s *fs, char *validsectors, int
 /****************************************************************************
  * Name: smartfs_recover
  *
- * Description: Recovery after  a power failure
+ * Description: Recovery after a power failure
```

Comment:
How about replacing 'Recovery' with 'Recover'?
Other description starts with a verb.

Reply:
ok, please do some more at the volley, add more information (recovery what? ; sector which has broken link)

Reply:
Modify. Thank you for your feedback.